
Disable strange BUG_ON's/drop added by AVM in Fritz!OS-07.0X

--- linux-3.10/net/core/dev.c
+++ linux-3.10/net/core/dev.c
@@ -3701,7 +3701,7 @@
 {
 	int ret;
 
-	BUG_ON(skb->sk);
+//	BUG_ON(skb->sk);
 	if (sk_memalloc_socks() && skb_pfmemalloc(skb)) {
 		unsigned long pflags = current->flags;
 
@@ -3742,7 +3742,7 @@
 {
 	int ret;
 
-	BUG_ON(skb->sk);
+//	BUG_ON(skb->sk);
 	net_timestamp_check(netdev_tstamp_prequeue, skb);
 
 	if (skb_defer_rx_timestamp(skb))
--- linux-3.10/net/ipv4/ip_forward.c
+++ linux-3.10/net/ipv4/ip_forward.c
@@ -134,8 +134,8 @@
 	struct rtable *rt;	/* Route we use */
 	struct ip_options *opt	= &(IPCB(skb)->opt);
 
-	if (unlikely(skb->sk))
-		goto drop;
+//	if (unlikely(skb->sk))
+//		goto drop;
 
 	if (skb_warn_if_lro(skb))
 		goto drop;
--- linux-3.10/net/ipv6/ip6_output.c
+++ linux-3.10/net/ipv6/ip6_output.c
@@ -395,8 +395,8 @@
 	if (net->ipv6.devconf_all->forwarding == 0)
 		goto error;
 
-	if (unlikely(skb->sk))
-		goto drop;
+//	if (unlikely(skb->sk))
+//		goto drop;
 
 	if (skb_warn_if_lro(skb))
 		goto drop;
